tools/syz-declextract/testdata/scopes.c

```diff
@@ -5,13 +5,19 @@
 #include "include/syscall.h"
 #include "include/uapi/file_operations.h"
 
+#define LARGE_UINT (1ull<<63)
+#define LARGE_SINT (20ll<<63)
```

Comment:
It would help to have some comment re this value. `20ll<<63` overflows 64 bit integers, is that the intention?